openstf_dbc/services/predictor.py

```diff
@@ -89,11 +89,8 @@ def get_market_data(self, datetime_start, datetime_end, forecast_resolution=None
         electricity_price = self.get_electricity_price(
             datetime_start, datetime_end, forecast_resolution
         )
-        gas_price = self.get_gas_price(
-            datetime_start, datetime_end, forecast_resolution
-        )
 
-        return pd.concat([electricity_price, gas_price], axis=1)
+        return pd.concat([electricity_price], axis=1)
```

Comment:
I think `pd.concat` is not necessary, `electricity_price` is already a `pd.DataFrame`. So, just returning `electricity_price`, should be enough.

Reply:
Agreed